pkg/events/registry.go

```diff
@@ -0,0 +1,37 @@
+package events
+
+import (
+	"k8s.io/klog"
+)
+
+type EventRegistry struct {
+	eventHandlers map[string]EventHandler
+	name          string
+}
+
+func NewRegistry(name string) EventRegistry {
+	return EventRegistry{
+		name: name,
```

Comment:
`eventHandlers` also needs to be initialized.

Reply:
Right, This part of the registry I think it's probably better handled in a separate PR, with it's own unit testing and all..